tutorials/fundamentals/undefined-check.md

```diff
@@ -0,0 +1,29 @@
+In JavaScript, checking if a variable is undefined can be a bit tricky since
+a [null](/tutorials/fundamentals/null) variable can pass a check for undefined if not written properly.
+As a result, this allows for `undefined` values to slip through and vice versa.
+This is because `null` and `undefined` both mean nothing, so checking if they
```

Comment:
Tutorials get a little messy when they go into semantics like what `null` and `undefined` mean. I'd just replace the last 2 sentences with "Make sure you use strict equality `===` to check if a value is equal to `undefined`"